### DISCUSSION TITLE
How should this library be imported when using es modules?

### DISCUSSION THREAD

**Discussion by alexbjorlig** [oozcitak/xmlbuilder2#92] [Q&A]
I seem to get the error: 

```
ENOENT: no such file or directory, open '<full-path>/node_modules/xmlbuilder2/src/index.ts'
```
when importing with: 
```
import { create } from 'xmlbuilder2';
```

**Comment by oozcitak** [comment]
Your `package.json` must include `type`:
```json
{
  "dependencies": {
    "xmlbuilder2": "2.4.1"
  },
  "type": "module"
}
```
Follow by a `npm i`. Then the import should work:
```js
import { create } from "xmlbuilder2";

const str = create({ version: "1.0" }).ele("root").end();
console.log(str);
```
Here is a minimum sandbox with the above: https://codesandbox.io/s/brave-spence-n1cor?file=/package.json

Your error message seems to indicate that `xmlbuilder2` is not installed. Did you `npm i`?
